### ISSUE TITLE
SmartBanner is in wrong position - At top instead of bottom

### ISSUE THREAD

**Issue by KhmerRun** [googleads/googleads-mobile-unity#558]
Hello,

I have an issue with my phone which the smart banner is in the wrong position. Briefly:
- The SmartBanner was in a correct position when I use version 3.7. Everything was fine.
- After upgrade the package to 3.8, compiled and testing it, the SmartBanner show at the top instead of the bottom for my mobile (Mi Max) but show correctly with my tab (Mi Pad 2).
- I didn't modify anything besides upgrading the package downloaded from Github.

Compiled with Unity3d 5.6.1.

**Comment by SnnYsf** [comment]
same problem . v3.7.1  v3.8.0.

v3.7.0 not have. working

**Comment by sonicsmith** [comment]
I have the same problem, and it happens even with non-smart banners.

**Comment by rampara** [comment]
@KhmerRun @SnnYsf @sonicsmith can you list all the devices you're able to reproduce this issue with. All of the devices I've tried render the banner in the expected position.

**Comment by sonicsmith** [comment]
Samusung S8.

**Comment by sonicsmith** [comment]
FYI: I was able to still change the position of the banner using custom values.

**Comment by KhmerRun** [comment]
@rampara 
I tested only with two android devices - Mi Max and Mi Pad 2. Also tried with Android Emulator.
- Mi Max: Show banner at the top (Android 7, MIUI 8.5)
- Mi Pad: Show banner at the bottom (correct)
- Android Emulator (various android version and screen): Show banner at the bottom (correct)

**Comment by rampara** [comment]
@sonicsmith @KhmerRun looks like this was caused by an Android 7 specific PopUpWindow bug. Will be pushing out a new release shortly that includes a fix.

**Comment by sonicsmith** [comment]
Thanks @rampara !

**Comment by SnnYsf** [comment]
Hi
The user reporting the problem is using android 7.
Thanks

**Comment by kangoo13** [comment]
@rampara  thanks for the fix, waiting for it :)

**Comment by rchallenger** [comment]
Got same trouble with Small Banner 320x50.
We load banner each time it has to be in different position after scene loaded and hide banner before scene load started. We does not change its position in current scene. But after scene is loaded we request new banner with certain position (in most cases it is bottom_centered). Banner loads in few seconds after request and appears in a right place. But some time later something happens and it changes its position dynamically to top_left. Here is a [video ](https://drive.google.com/open?id=1EwB6tuTL6HovXjoOUoh0tR-wSupktBo1)of what i was talking about.

We used GAD 3.8.0, play-services-11.6.0, Unity 5.4.0.

PS: Even if its position was changed from code there is no other way except destroying old banner view and creating new with different position argument. So it could be seen as banner disappearing and appearing in few seconds, after new ad will be loaded.
